paypal.js

```diff
@@ -92,6 +93,11 @@ var detail_q = async.queue(function(task, next) {
     }
 
     task.COUNTRYCODE = transaction.COUNTRYCODE;
+    task.settle_amount = transaction.AMT;
+    if (task.CURRENCYCODE && task.CURRENCYCODE !== 'USD' && transaction.EXCHANGERATE) {
+      task.settle_amount = (transaction.AMT * transaction.EXCHANGERATE * 100 | 0 ) / 100;
```

Comment:
Why add the `| 0` in there? could transaction.AMT ever be `undefined` or something?